### DISCUSSION TITLE
Weird AI bots

### DISCUSSION THREAD

**Discussion by PROJEKT-D3K** [kus/cs2-modded-server#126] [Q&A]
My server's bots only uses knife and AWP(if they dont have awp,they just stab each other)
Is this really normal? 
PS.I've checked the bot.cfg already and both bot_pistols_only and bot_sniper_only are set to 0

**Comment by GooberOTF** [comment]
Please let me know if you figure this out. I was having the same issue..

**Comment by GooberOTF** [comment]
> My server's bots only uses knife and AWP(if they dont have awp,they just stab each other) Is this really normal? PS.I've checked the bot.cfg already and both bot_pistols_only and bot_sniper_only are set to 0

did you ever figure this out? I have tried editing alot too to see if i can get them to not use Awps / knifes only lol! 

Thanks!
